CardFlip.js

```diff
@@ -235,36 +235,37 @@ class CardFlip extends Component<Props> {
     const sideBOpacity = progress.interpolate({
       inputRange: [50, 51],
       outputRange: [0, 100],
-      extrapolate: "clamp"
+      extrapolate: 'clamp',
     });
 
     const sideBTransform = {
       opacity: sideBOpacity,
       zIndex: side === 0 ? 0 : 1,
-      transform: [{ perspective: -1 * this.props.perspective }]
+      transform: [{ perspective: -1 * this.props.perspective }],
     };
 
-    if (rotateOrientation === "x") {
+    if (rotateOrientation === 'x') {
       const bXRotation = rotation.x.interpolate({
         inputRange: [0, 50, 100, 150],
-        outputRange: ["0deg", "-180deg", "-360deg", "180deg"],
-        extrapolate: "clamp"
+        outputRange: ['0deg', '-180deg', '-360deg', '180deg'],
+        extrapolate: 'clamp',
       });
       sideBTransform.transform.push({ rotateX: bXRotation });
     } else {
-      if (Platform.OS === "ios") {
+      let bYRotation;
```

Comment:
Maybe this can be done with ternary operator?

Reply:
Definitely. There's probably some other refactoring that can be done as well, but I was just hoping to get this bug quickly fixed in this PR.